Enterprise-CMCS/macpro-mdct-mfp
# sar.json Initial Buildout

### Description

Getting a head start on the SAR form pages.

### Related ticket(s)


MDCT-2963

---
### How to test

- Navigate to `/sar`
- Generate a new SAR report
- Go through the pages, make sure everything looks good

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
